Test/Tasty/Golden/Advanced.hs

```diff
@@ -32,4 +33,34 @@ goldenTest
   -> (a -> IO ())
     -- ^ update the golden file
   -> TestTree
-goldenTest t golden test cmp upd = singleTest t $ Golden golden test cmp upd
+goldenTest t golden test cmp upd = singleTest t $ Golden golden test cmp upd (return ())
+
+-- | A variant of 'goldenTest' that also provides for deleting the output
+-- file.
```

Comment:
Adding a mention of the `DeleteOutputFile` option could also be helpful here, to give an explanation why this variant exists.

Reply:
Done.